src/event_handler.py

```diff
@@ -18,33 +18,46 @@ def store_queued_events(_, __):
     sqs_client = boto3.client('sqs')
     queue_url = os.environ['QUEUE_URL']
 
+    logger = logging.getLogger('event-recorder')
     if 'ENCRYPTION_KEY' in os.environ:
       encrypted_decryption_key = os.environ['ENCRYPTION_KEY']
+      logger.info('Got decryption key from envrionment variable')
     else:
       encrypted_decryption_key = fetch_decryption_key()
+      logger.info('Got decryption key from S3')
     decryption_key = decrypt(encrypted_decryption_key)
+    logger.info('Decrypted key successfully')
 
     database_password = None
     if 'ENCRYPTED_DATABASE_PASSWORD' in os.environ:
       # boto returns decrypted as b'bytes' so decode to convert to password string
       database_password = decrypt(os.environ['ENCRYPTED_DATABASE_PASSWORD']).decode()
     db_connection = create_db_connection(database_password)
+    logger.info('Created connection to DB')
 
+    event_count = 0
     while True:
+        event_count += 1
```

Comment:
Should this be moved below line 44? Otherwise we can never exit after 0 events...

Reply:
Well spotted 👀